MultiSnap_timelapse_ef_230209_3.bsh

```diff
@@ -254,6 +257,8 @@ for(int qy=0;qy<panelnumy;qy++){
 
 		// %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 		// %%%%%%  don't acquire if condition flag is not 1		
+		time_indexH = time_indexH + 1;	//check�K�v
```

Comment:
did this work?

Reply:
it is worked for image which showed on display.
However, saved data had same problem as I reported in last WR.
I will exprain in detail at WR.